src/main/java/teammates/common/util/JsonUtils.java

```diff
@@ -23,8 +32,7 @@ private JsonUtils() {
      * Json file and also reformat the Json string in pretty-print format.
      */
     private static Gson getTeammatesGson() {
-        return new GsonBuilder().setDateFormat(DateFormat.FULL)
-                                .setDateFormat(Const.SystemParams.DEFAULT_DATE_TIME_FORMAT)
+        return new GsonBuilder().registerTypeAdapter(Date.class, new StandardDateAdapter())
```

Comment:
There's no actual change to the expected format - `Const.SystemParams.DEFAULT_DATE_TIME_FORMAT` is still used. `setDateFormat(DateFormat.FULL)` previously didn't do anything as it was overwritten by `.setDateFormat(Const.SystemParams.DEFAULT_DATE_TIME_FORMAT)`.